### PR TITLE
Added option to show user's last submissions

### PR DESCRIPTION
**Pull request by gabrielsimoes**
I'll still improve it, but it works. Emacs automatically fixed indentation on files.

### CODE REVIEW

File: cf-mode.el [hunk 1]
```diff
@@ -50,42 +50,59 @@
...
-	   (cons (kbd "C-c c t") 'cf-test-all-i)
-	   (cons (kbd "C-c c d") 'cf-download-tests-i)))
+           (cons (kbd "C-c c w") 'cf-whoami-i)
+           (cons (kbd "C-c c s") 'cf-submit-current-buffer-by-path-i)
+           (cons (kbd "C-c c i") 'cf-login-i)
+           (cons (kbd "C-c c o") 'cf-logout-i)
+           (cons (kbd "C-c c w") 'cf-whoami-i)
+           (cons (kbd "C-c c t") 'cf-test-all-i)
+           (cons (kbd "C-c c d") 'cf-download-tests-i)
+           (cons (kbd "C-c c l") 'cf-last-submissions-i))
```
Comment (gnull) [inline cf-mode.el]: Looks like here must be closing bracket for define-minor-mode.

Comment (gabrielsimoes) [inline cf-mode.el]: Sure, fixed it.

**Comment by gnull** [comment]
I have already fixed that bracket and indentation after it. Also added (require 'json)

**Comment by gnull** [comment]
The changes are at the master branch!

**Pull request closed without merging**